google/cloud/texttospeech_v1/services/text_to_speech/client.py

```diff
@@ -228,11 +236,11 @@ def list_voices(
                 language tag. If specified, the ListVoices call will
                 only return voices that can be used to synthesize this
                 language_code. E.g. when specifying "en-NZ", you will
-                get supported "en-\*" voices; when specifying "no", you
-                will get supported "no-\*" (Norwegian) and "nb-*"
+                get supported "en-*" voices; when specifying "no", you
+                will get supported "no-*" (Norwegian) and "nb-*"
                 (Norwegian Bokmal) voices; specifying "zh" will also get
                 supported "cmn-*" voices; specifying "zh-hk" will also
-                get supported "yue-\*" voices.
+                get supported "yue-*" voices.
```

Comment:
I'm not sure why the synth replace isn't working for this, but this is the reason for the docs failure.

Reply:
Yeah, the autosynth PR also has the same thing:

https://github.com/googleapis/python-texttospeech/pull/70/files#diff-b794d7ad2724dfc5c6821d8d0edb1bebR239

Reply:
I think I figured it out:

```diff
-    "((en)|(no)|(nb)(cmn)|(yue))-\*",
+    "((en)|(no)|(nb)|(cmn)|(yue))-\*",

```

Reply:
Now it's green!

Reply:
Woo! Thank you! 😁